pageserve/test_routes.py

```diff
@@ -350,6 +350,13 @@ def test_add_invalid_post(self, mock_requests):
         self.assertEqual(response.status_code, 400)
         self.assertEqual(response.data.decode(), 'Example error message')
 
+    def test_not_logged_in(self):
+        """Not logged in, don't add."""
+        response = self.client.post('/v1/add_post', data=INVALID_POST_FORM)
```

Comment:
I think it would be better to use `VALID_POST_FORM` here so you're only testing a missing user.

Reply:
+1

Reply:
Thanks for the catch, done https://github.com/knative-portability/large-events/pull/70/commits/14517d48654c7704dd2675d199c0942eef42f66d